src/rp2040/hardware_structs/include/hardware/structs/interp.h

```diff
@@ -22,21 +22,52 @@
 
 typedef struct {
     _REG_(SIO_INTERP0_ACCUM0_OFFSET) // SIO_INTERP0_ACCUM0
+    // (Description copied from SIO_INTERP0_ACCUM0 applies similarly all array members)
+    //
+    // Read/write access to accumulator 0
     io_rw_32 accum[2];
 
     _REG_(SIO_INTERP0_BASE0_OFFSET) // SIO_INTERP0_BASE0
+    // (Description copied from SIO_INTERP0_BASE0 applies similarly all array members)
+    //
+    // Read/write access to BASE0 register
     io_rw_32 base[3];
 
     _REG_(SIO_INTERP0_POP_LANE0_OFFSET) // SIO_INTERP0_POP_LANE0
+    // (Description copied from SIO_INTERP0_POP_LANE0 applies similarly all array members)
+    //
+    // Read LANE0 result, and simultaneously write lane results to both accumulators (POP)
     io_ro_32 pop[3];
```

Comment:
Trying to match this up to the RP2040 datasheet, it's slightly confusing here that `pop[0]` refers to `INTERP0_POP_LANE0`, `pop[1]` refers to `INTERP0_POP_LANE1` but `pop[2]` refers to `INTERP0_POP_FULL`.
However I guess there's not really much that can be done about that in an automated manner.